free_fleet_adapter/free_fleet_adapter/nav2_robot_adapter.py

```diff
@@ -119,6 +121,9 @@ def __init__(
         self.nav_goal_id = None
         self.map_name = self.robot_config_yaml['initial_map']
 
+        self.frame_map = self.robot_config_yaml['map_frame']
+        self.frame_robot = self.robot_config_yaml['robot_frame']
```

Comment:
Check if `self.robot_config_yaml` does indeed contain `map_frame` or `robot_frame`, otherwise use default values. This will help to prevent breaking existing behavior for other configs